File: edu/HRO/CMITDW Taal der Wiskunde/o71.java
// opdracht 7.1; puzzel 1
public class o71
{
	public static void main(String[] args)
	{
		System.out.println("/5\tsquare\tprime");
		
		for (int i=1; i<=1000; i++)
		{
			if (i%5 == 0) // dividable by 5
			{
				if (Math.sqrt(i+1)%1 == 0) // quadratic
				{
					boolean isprime = true;
					for (int j=2; j<i+2; j++)
					{
						if ((i+2)%j == 0) // not prime
						{
							isprime = false;
							break;
						}
					}
					if (isprime)
						System.out.println(i+"\t"+(i+1)+"\t"+(i+2));
				}
			}
		}
	}
}

File: edu/HRO/CMITDW Taal der Wiskunde/Lua/o71.lua
-- opdracht 7.1; puzzel 1
print("/5", "square", "prime")

for i = 1, 1000 do
	if i%5 == 0 then -- dividable by 5
		if math.sqrt(i+1)%1 == 0 then -- quadratic
			local isprime = true
			for j = 2, i+1 do
				if (i+2)%j == 0 then -- not prime
					isprime = false
					break
				end
			end
			if isprime then
				print(i, i+1, i+2)
			end
		end
	end
end
